org-formation/organization.yaml

```diff
@@ -248,3 +249,13 @@ Organization:
       Tags:
         Department: Platform
         Project: Infrastructure
+
+  NlpSandboxAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: NlpSandboxAccount
+      RootEmail: support@nlpsandbox.io
```

Comment:
This should be an email address in the sagebase.org domain

Reply:
i agree with @ahayden.  Just tell us the email you'de like to use and we will create it.